google/api_core/exceptions.py

```diff
@@ -476,22 +476,31 @@ def from_http_status(status_code, message, **kwargs):
     return error
 
 
-def from_http_response(response):
-    """Create a :class:`GoogleAPICallError` from a :class:`requests.Response`.
+def _format_error_message(error, method, url):
+    method = method.upper()
+    message = "{method} {url}: {error}".format(
+        method=method,
+        url=url,
```

Comment:
These parameters are REST-specific, so maybe the name of this function should include `_rest_` to make that explicit.

Reply:
addressed.